pricegraph/src/orderbook/user.rs

```diff
@@ -11,49 +11,47 @@ pub type UserMap = Map<UserId, User>;
 #[derive(Clone, Debug, Default)]
 pub struct User {
     /// User balances per token.
-    balances: Map<TokenId, f64>,
+    balances: Map<TokenId, u128>,
```

Comment:
Technically user balances are U256. We round too large balances to u128::MAX. In practice I don't think this is a problem because individual trades are limited to u128::MAX.

Reply:
This should probably be a `U256` or even `BigInt` since the balance can be used for multiple orders.

Reply:
Changed it to U256.